README.md

```diff
@@ -322,6 +322,48 @@ class PostSerializer < ActiveModel::Serializer
 end
 ```
 
+## Serializable Resource Interface
+
+Serializable resources must include [`ActiveModel::Serialization`](https://github.com/rails/rails/blob/master/activemodel/lib/active_model/serialization.rb)
+or implement its interface:
+
+- `#serializable_hash(options)`: hash representation of a resources attributes
+- `#read_attribute_for_serialization(name)`: gets the attribute value for serialization
+
+ActiveRecord::Base objects include ActiveModel::Serialization
+
+The additionally must implement:
+- `#as_json(options)`: Hash representation of a serialized resource, may delegate to #serializable_hash
+- `#to_json(options)`: string representation (JSON) to Hash from `as_json`
+- `#cache_key`: a (self-expiring) unique key for the instance, used by the adapter
+- `#id`: a unique identifier for the object
+- `::model_name`: an ActiveModel::Name instance, used by the serializer to identify the type
+
+ActiveRecord::Base objects implement these methods by default.
+
+## Manual Serialization
+
+For the simple case, you may serialize a object with the below snippet:
+
+```ruby
+serializer_class = ActiveModel::Serializer.serializer_for(obj)
+serializer = serializer_class.new(obj)
+adapter = ActiveModel::Serializer.adapter.new(serializer)
+adapter.serializable_hash
```

Comment:
Yeah, it's been needing an update for a few months.. I just haven't circle back to it